go.mod

```diff
@@ -7,6 +7,8 @@ require (
 	github.com/google/go-cmp v0.5.6
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.15.0
+	github.com/openshift/api v3.9.0+incompatible
```

Comment:
can you try with below?
``` diff
diff --git a/go.mod b/go.mod
index ddfee36..613087f 100644
--- a/go.mod
+++ b/go.mod
@@ -7,8 +7,8 @@ require (
        github.com/google/go-cmp v0.5.6
        github.com/onsi/ginkgo v1.16.4
        github.com/onsi/gomega v1.15.0
-       github.com/openshift/api v3.9.0+incompatible
-       github.com/openshift/client-go v3.9.0+incompatible
+       github.com/openshift/api v0.0.0-20211028023115-7224b732cc14
+       github.com/openshift/client-go v0.0.0-20210831095141-e19a065e79f7
        github.com/topolvm/topolvm v0.10.3
        gotest.tools/v3 v3.0.3
        k8s.io/api v0.22.2
```
- above is the result of
```
->  go get github.com/openshift/api@release-4.9
-> go get github.com/openshift/client-go@release-4.9
```

Reply:
That works. Thanks.